packages/manager/apps/pci-kubernetes/src/api/data/kubernetes.ts

```diff
@@ -114,3 +114,21 @@ export const terminateCluster = async (projectId: string, kubeId: string) => {
   );
   return data;
 };
+
+export type ResetClusterParams = {
```

Comment:
```suggestion
export type TResetClusterParams = {
```
nitpick for consistency